Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/callback/opensaml/CertificateManagerImplTest.java

```diff
@@ -107,4 +108,34 @@ public void testGetDefaultCertificate() throws Exception {
         assertEquals(certificate.getSigAlgName(), "SHA256withRSA");
         assertEquals(certificate.getSigAlgOID(), "1.2.840.113549.1.1.11");
     }
+
+    @Test
+    public void testCertificateChain() throws Exception {
+        KeyStore keystore = CertificateUtil.loadKeyStore("JKS", "changeit", "src/test/resources/cacerts-chain.jks");
```

Comment:
Why you are specifying the path here? If you run mvn clena install, the cacerts-chain.jks will be copied to target folder and the maven will pick it up from there